website/docs/cloud-docs/projects/managing.mdx

```diff
@@ -0,0 +1,82 @@
+---
+page_title: Managing Projects
+description: |-
+  Use projects to organize and group workspaces and create ownership boundaries
+  across your infrastructure.
+---
+
+# Managing Projects
+
+Projects let you organize your workspaces and scope access to workspace
+resources.
+
+## Permissions
+
+You must be a member of a team with the **Manage all Projects**
+[organization-level
+permissions](/terraform/cloud-docs/users-teams-organizations/permissions#organization-permissions)
+permissions to create a project.
+
+You must be a member of a team with the update team access to add a new team
+access for organization-visible teams for a project. 
+
+You must be a member of a team with update and delete permissions to be able to
+update and delete teams respectively.
+
+## View a Project
```

Comment:
It feels a bit strange to have the "View Projects" and the "View a project" sections separated on two different pages. How would you feel about moving the "View projects" section to this page too?